server/gateway/src/alfred.ts

```diff
@@ -3,7 +3,7 @@
  * Licensed under the MIT License.
  */
 
-import { IFluidCodeDetails } from "@fluidframework/container-definitions";
```

Comment:
nothing under server is part of the client mono-repo, we version bump this after release, so these should all be reverted for now